src/libAtomVM/nifs.gperf

```diff
@@ -133,6 +133,7 @@ atomvm:posix_select_write/3, IF_HAVE_OPEN_CLOSE(&atomvm_posix_select_write_nif)
 atomvm:posix_select_stop/1, IF_HAVE_OPEN_CLOSE(&atomvm_posix_select_stop_nif)
 atomvm:posix_mkfifo/2, IF_HAVE_MKFIFO(&atomvm_posix_mkfifo_nif)
 atomvm:posix_unlink/1, IF_HAVE_UNLINK(&atomvm_posix_unlink_nif)
+atomvm:posix_clock_settime/2, &atomvm_posix_clock_settime_nif
```

Comment:
I am uncertain, but shouldn't this be `IF_HAVE_CLOCK_SETTIME(&atomvm_posix_clock_settime_nif)`, or some similar conditional here?

Reply:
No, because the function will return `undef` on platforms that don’t support it (in this specific implementation)